webapp/redux/middleware/middleware.js

```diff
@@ -106,6 +106,33 @@ const simulateNetwork = (payload) => createSimulateBackendCall(
   GEPPETTO.Resources.RUNNING_SIMULATION,
 );
 
+class ErrorDialogManager {
```

Comment:
I'm not convinced by the implementation of the ErrorDialogManager, starting from the name. It does not manage any dialog, so the name is misleading. I would rather call it PythonMessageClassifier or something similar

Reply:
I don't see what's being classified here but let's do something in the middle and call it PythonMessageFilter maybe?